### PR TITLE
feat: 'Resources' tab in the navigation menu and sign up form in Pillar pages 

### PR DESCRIPTION
**Pull request by Hermione2408**
- Added 'Join Waitlist' form in the Features section.
- Introduced a 'Resources' tab in the navigation menu.
- Commented Problem block in landing page.
- Added testimonial right before the community section on the homepage.
Screenshot from 2024-04-26 22-11-40
Screenshot from 2024-04-26 22-13-57
Screenshot from 2024-04-26 22-23-49
Screenshot from 2024-04-26 22-23-31

### CODE REVIEW

**Comment by Sonichigo** [comment]
this PR also resolve [#1842](https://github.com/keploy/keploy/issues/1842)

File: components/nav/navItemWithSmallDropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import React, {useState} from "react";
+import Link from "next/link";
+import {UpIcon} from "./UpIcon"
+import DownIcon from "./DownIcon"
+import Image from "next/image";
+
+export interface NavItemWithSmallDropdownProps {
```
Comment (Sonichigo) [inline components/nav/navItemWithSmallDropdown.tsx:7]: image

The dropdown overlay with the hero in terms of background, i think it's better to keep the dropdown background either with light grey or `#E6E2D4` color or when dropdown menu is clicked rest background get's a bit darker to put focus on resources section

Comment (Hermione2408) [reply components/nav/navItemWithSmallDropdown.tsx:7]: Hey @Sonichigo, I've updated the dropdown background to light grey to ensure it contrasts well with background. This should help in focusing on the resources section when the dropdown is active. Can you take a look?

**Comment by Sonichigo** [comment]
LGTM!

**Review by nehagup — CHANGES_REQUESTED** [review]
- update form registration message - We'll get in touch with you shortly
- correct alignment on resource tab
- join the waitlist heading - on the form. 
- if text area is expanded, the form breaks with no-scrolling capability.

**Comment by Hermione2408** [comment]
> * update form registration message - We'll get in touch with you shortly
> * correct alignment on resource tab
> * join the waitlist heading - on the form.
> * if text area is expanded, the form breaks with no-scrolling capability.

Hey @nehagup I have pushed the fixes, Please have a look!

**Review by nehagup — APPROVED** [review]

**Pull request merged**